spec/requests/ping_spec.rb

```diff
@@ -0,0 +1,8 @@
+describe "PingController" do
+  it "get" do
+    get("/api/ping")
```

Comment:
minor, could you update to use the rails helper for consistency with the other specs.

```
run_get api_ping_url
```

Thanks,